creator-node/src/routes/files.js

```diff
@@ -550,44 +546,30 @@ const getDirCID = async (req, res) => {
   // Do not act as a public gateway. Only serve files that are tracked by this creator node.
   const dirCID = req.params.dirCID
   const filename = req.params.filename
-  const path = `${dirCID}/${filename}`
-  const logPrefix = '[getDirCID]'
-
-  const cacheKey = getStoragePathQueryCacheKey(path)
-
-  let storagePath = await redisClient.get(cacheKey) // can be overridden later by computed storage location if cache doesn't exist
-  let CID = null // this gets assigned later after the correct storagePath is set
-  let queryResults
-
-  if (!storagePath) {
-    // We need to lookup the CID so we can correlate with the filename (eg original.jpg, 150x150.jpg)
-    // Query for the file based on the dirCID and filename
-    queryResults = await models.File.findOne({
-      where: {
-        dirMultihash: dirCID,
-        fileName: filename
-      },
-      order: [['clock', 'DESC']]
-    })
-    if (!queryResults) {
-      return sendResponse(
-        req,
-        res,
-        errorResponseNotFound(
-          `No valid file found for provided dirCID: ${dirCID} and filename: ${filename}`
-        )
-      )
-    }
-
-    storagePath = DiskManager.computeFilePathInDir(
-      dirCID,
-      queryResults.multihash
+  const logPrefix = `[getDirCID][dirCID: ${dirCID}][fileName: ${filename}]`
+
+  // We need to lookup the CID so we can correlate with the filename (eg original.jpg, 150x150.jpg)
```

Comment:
all this is same code from inside the `if (!storagePAth)` on the left since all that wrapper code is removed